src/validation/config.json

```diff
@@ -19,15 +19,24 @@
           "controls": false,
           "compass": false,
           "coordinates": false
-        }
+        },
+        "defaultLayers": [
```

Comment:
The 'defaultLayers' object is not necessary but does function if a different WMS service or layer is specified. If none are listed WorldWind will continue to use the default BlueMarble and reach back to NASA for the imagery.